source/core/src/main/com/csse3200/game/components/animal/AirAnimalSelectionDisplay.java

```diff
@@ -0,0 +1,28 @@
+package com.csse3200.game.components.animal;
+
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+
+/**
+ * Class representing the air animal selection screen.
+ */
+public class AirAnimalSelectionDisplay extends AnimalSelectionDisplay {
+
+    public AirAnimalSelectionDisplay(Stage stage, Skin skin) {
+        super(stage, skin);
```

Comment:
Is it not possible to set the stage in AnimalSelectionScreen and use the ServiceLocator? Or better might be to set the stage in animal selection screen (see how main game screen sets up a render service) and then have this extend UI component (maybe just change UI component to not have a static skin and just set a default, and just provide a function to set a new skin)?